documentation/api/parameters.md

```diff
@@ -1,12 +1,12 @@
 
 ### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Fapi%2Fparameters)
 
-# Parameters - Post (experimental feature)
+# Parameters (experimental feature)
 
-Captures parameters for one or more methods each time they are called. Parameters are logged inside the target application using its [`ILogger`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger).
+Captures parameters for one or more methods each time they are called.
 
 > [!NOTE]
-> Unlike other artifacts, parameters do **not** support being sent to an egress provider.
+> Before version 9.0 Preview 3, parameters were logged inside the target application using its [`ILogger`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger).
```

Comment:
```suggestion
> Before version 9.0 Preview 4, parameters were logged inside the target application using its [`ILogger`](https://learn.microsoft.com/dotnet/api/microsoft.extensions.logging.ilogger).
```

Reply:
Oops, I forgot about that. I've completly removed the note since the team decided not to mention the old behavior in documentation.